src/dnddiceroller/DnDDiceRoller.java

```diff
@@ -38,12 +39,12 @@ public static int test_oneD6(){
   }
   
   public static int test_diceTowerWithTwoD6(){
-	List<Die> dice = new ArrayList();
-	dice.add(new Die(6));
-	dice.add(new Die(6));
-	DiceTower tower = DiceTower(dice);
+	List<NumberedDie> dice = new ArrayList();
```

Comment:
Die should still be fine because All Die follow the same rules as far as expected methods